terraform/modules/client-elbs/star_18f_gov_elb.tf

```diff
@@ -1,4 +1,6 @@
 resource "aws_elb" "star_18f_gov_elb" {
+  count = "${var.count}"
```

Comment:
Instead of using a `var` here, WDYT about using a conditional based on `stack_description`?
https://www.terraform.io/docs/configuration/interpolation.html#conditionals